### PR TITLE
Only keep changes to a request if they're saved

### PR DESCRIPTION
**Pull request by forevermatt**

### CODE REVIEW

File: src/views/CreateOrUpdateRequest.svelte [hunk 1]
```diff
@@ -18,20 +18,26 @@ export let params = {} // URL path parameters, provided by router.
 
 let imageUrl = ''
 
-const newRequest = {
+const defaultRequest = {
```
Comment (forevermatt) [inline src/views/CreateOrUpdateRequest.svelte]: @wcjr - On second thought, naming this `defaultRequest` isn't sitting well in my mind. When I see "defaultRequest", it sounds like some other (complete) request that we intend to show if we weren't given a request. However, this is just to provide default values for when we're creating a new request.

What would you think about calling it `newRequestDefaults` or `defaultRequestValues` instead?

Comment (wcjr) [reply src/views/CreateOrUpdateRequest.svelte]: I'd be happy with `defaults` but go with what will ease your mind the most.

Comment (wcjr) [reply src/views/CreateOrUpdateRequest.svelte]: I appreciate wanting to get that little nuisance out of there :-) Makes me glad you love what you do.

Comment (wcjr) [reply src/views/CreateOrUpdateRequest.svelte]: if I had to choose between your options, I like `defaultRequestValues` the best.

**Pull request merged**